db/seeds/development/comments_seed.rb

```diff
@@ -1,13 +1,20 @@
-def generate_comment(student)
+def generate_comment(id, type)
   new_comment = Comment.create(
+    commentable_id: "#{id}",
+    commentable_type: type,
     content: "#{Faker::Lorem.sentence}",
-    student: student,
     user_id: 1 + rand(4),
   )
 end
 
 Student.all.each do |student|
-  generate_comment(student)
-  generate_comment(student)
+  generate_comment(student.id, "Student")
+  generate_comment(student.id, "Student")
```

Comment:
Let's use single quotes here, we only use double quotes when we need to use string interpolation (`#{}`).